exercises/simple-cipher/simple-cipher.spec.js

```diff
@@ -26,21 +26,15 @@ describe('Random key cipher', function () {
 
 describe('Incorrect key cipher', function () {
   xit('throws an error with an all caps key', function () {
-    expect( function () {
-      new Cipher('ABCDEF');
-    }).toThrow(new Error('Bad key'));
+    expect(new Cipher('ABCDEF')).toThrow(new Error('Bad key'));
```

Comment:
You should revert this change.  The `toThrow` matcher requires a function as an argument, but your change provides it with a `Cipher` object instead.  

I imagine you were seeing an error for the eslint `no-new` rule.  You can disable for this test block with:
```javascript
  /* eslint-disable no-new */
describe('Incorrect key cipher', function () {
...
});
  /* eslint-enable no-new */
```